src/responseHandlers/TestResponseHandler.js

```diff
@@ -91,7 +91,7 @@ var TestResponseHandler = jsface.Class(AbstractResponseHandler, {
         sweet += "for(p in sugar.array)  {if(p==='create'){Array.create=sugar.array.create} else{Array.prototype[p]= sugar.array[p];}}";
         sweet += "for(p in sugar.string) String.prototype[p]  = sugar.string[p];";
         sweet += "for(p in sugar.date)  {if(p==='create'){Date.create=sugar.date.create} else{Date.prototype[p]= sugar.date[p];}}";
-        sweet += "for(p in sugar.funcs)  Function.prototype[p]= sugar.funcs[p];";
+        sweet += "for(p in sugar.funcs) Object.defineProperty(Function.prototype, p, Object.getOwnPropertyDescriptor(sugar.funcs, p));";
```

Comment:
These will not work on older node. I think it must be checked before proceeding and then choose either.